### PR TITLE
Add extension packaging

### PR DESCRIPTION
**Pull request by uarlouski**

### CODE REVIEW

**Review by valfirst — APPROVED** [review]

**Review by ikalinin1 — APPROVED** [review]

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -43,3 +43,8 @@ jobs:
         uses: GabrielBB/xvfb-action@v1.5
         with:
           run: npm test
+      
+      - name: Extension packaging
+        run: |
+          sudo npm install -g vsce
+          vsce package
```
Comment (ikalinin1) [inline .github/workflows/build.yml:50]: please publish the extension to the gh packages

Comment (ikalinin1) [reply .github/workflows/build.yml:50]: ^^

**Review by ikalinin1 — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yml (hunk 1 above)
Comment (uarlouski) [reply .github/workflows/build.yml:50]: I will do this once releases are set up

Comment (uarlouski) [reply .github/workflows/build.yml:50]: gh packages doesn't support publishing of vsix applications

**Review by ikalinin1 — APPROVED** [review]

**Pull request merged**